tests/ram/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ add_xc7_test(
     tcl run.tcl
     sources ram.v
     testbench ram_tb.v
+    disable_vivado_test
```

Comment:
is this currently failing? do we know why?

Reply:
Actually this was a leftover but I need to double check it. What is failing is the validation test (due to https://github.com/SymbiFlow/python-fpga-interchange/issues/75).

Reply:
I have added also the ram test and it passed